lib/ansible/modules/windows/win_iis_website.py

```diff
@@ -92,6 +97,23 @@
     name: "Default Web Site"
     state: absent
 
+# Create a WebSite with custom Logging configuration (Logs Location, Format and Rolling Over).
+
+- name: Creating WebSite with Custom Log location, Format 3WC and rolling over every hour.
+      win_iis_website:
+       name: "MyCustom_Web_Shop_Site"
+       state: started
+       port: 80
+       ip: '*'
+       hostname: '*'
+       physical_path: D:\wwwroot\websites\my-shop-site
+       parameters: logfile.directory:D:\IIS-LOGS\websites\my-shop-site|logfile.period:Hourly|logFile.logFormat:W3C
+       application_pool: my-shop-site
+      register: dsc_state_is
```

Comment:
Maybe change the name from dsc to. Something else, it seems a bit out of place here